alembic/env.py

```diff
@@ -0,0 +1,80 @@
+from logging.config import fileConfig
+
+from sqlalchemy import engine_from_config
+from sqlalchemy import pool
+
+from alembic import context
+
+from models import Base
```

Comment:
Relative imports (.models) are typically used within the same package to avoid hardcoding the full path and to make the code more modular.